lib/cadet/accounts/query.ex

```diff
@@ -26,6 +27,15 @@ defmodule Cadet.Accounts.Query do
     |> where([_, g], g.leader_id == ^id)
   end
 
+  def is_avenger_of(avenger, student_id) do
```

Comment:
this technically isnt a query since its being executed. is there a better place to put this?

Reply:
Ah ok, I guess it's fine then.

Reply:
Also, do read the style guide at https://github.com/source-academy/cadet#style-guide. By convention, a function that returns boolean should not have `is_` in front, but rather end with question mark. so `def avenger_of?(avenger, student_id)` is better.